packages/iov-ethereum/src/serialization.ts

```diff
@@ -25,12 +24,17 @@ import { Abi } from "./abi";
 import { isValidAddress } from "./address";
 import { constants } from "./constants";
 import { BlknumForkState, Eip155ChainId, eip155V, toRlp } from "./encoding";
-import { Erc20Options } from "./erc20";
+import { Erc20ApproveTransaction, Erc20Options, Erc20TokensMap, isErc20ApproveTransaction } from "./erc20";
 import { encodeQuantity, encodeQuantityString, fromBcpChainId, normalizeHex } from "./utils";
 
 const { fromHex, toUtf8 } = Encoding;
 const ZERO_ETH_QUANTITY = "0";
 
+export enum SwapIdPrefix {
```

Comment:
I think I'd move this to the ID generation. But it should not matter much.

Reply:
That would create a circular dependency. We could move the ID generation here? Or to a separate file?

Reply:
Good point. Let's just leave it here for now.